includes/class.llms.template.loader.php

```diff
@@ -462,52 +516,62 @@ public function template_loader( $template ) {
 			 */
 			do_action( "llms_content_restricted_by_{$page_restricted['reason']}", $page_restricted );
 
-			// Blog should bypass checks, except when sitewide restrictions are enabled.
 			if ( is_home() && 'sitewide_membership' === $page_restricted['reason'] ) {
-
 				// Prints notices on the blog page when there's not redirects setup.
 				add_action( 'loop_start', 'llms_print_notices', 5 );
+			}
+		}
 
-				return $template;
+		$forced_template = apply_filters( 'llms_force_llms_template_loading', true ) ? $this->get_maybe_forced_llms_template() : false;
+		return $forced_template ? llms_template_file_path( $forced_template ) : $template;
 
-			}
+	}
 
-			// Course and membership content restrictions are handled by conditional elements in the editor.
-			if ( in_array( get_post_type(), array( 'course', 'llms_membership' ), true ) ) {
-				return $template;
+	/**
+	 * Retrieve the hierarchical template to be loaded.
+	 *
+	 * @since [version]
+	 *
+	 * @return null|string
+	 */
+	private function get_maybe_forced_llms_template() {
```

Comment:
```suggestion
	private function get_maybe_forced_template() {
```

Is `llms` necessary in our own internal private function?

Reply:
wasn't a prefixing choice, it was more a naming choice, a name that tells we're talking about llms specific templates.